generators/app/index.js

```diff
@@ -129,9 +129,9 @@ module.exports = class extends BaseGenerator {
             this.error(`\nYour backend project must be a monolith or a gateway to work with this module! Found application type: ${applicationType}.\n`);
         }
 
-        const cmd = `ionic start ${this.ionicAppName} oktadeveloper/jhipster${(this.interactive) ? '' : ' --no-interactive'}`;
+        const cmd = `ionic start ${this.ionicAppName} https://github.com/oktadeveloper/ionic-jhipster-starter${(this.interactive) ? '' : ' --no-interactive'}`;
```

Comment:
@pmverma 
The support for V4 is still not finished. We need to add entity support as V3 does.
So this change can't be added right now.
To fix this we can add the flag `--type=ionic-angular` for the users using Ionic v4

Reply:
Thanks @fleboulch. I am going to close this PR as I notice v4 support is already in progress.

Reply:
i will go to try this..